cartography/data/jobs/analysis/gcp_cloud_function_analysis.json

```diff
@@ -0,0 +1,18 @@
+{
+    "statements": [
+        {
+            "query": "MATCH (function:GCPFunction{exposed_internet:true})<-[:RESOURCE]-(:GCPProject{id: $GCP_PROJECT_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) WITH function LIMIT $LIMIT_SIZE REMOVE function.exposed_internet, function.exposed_internet_type return COUNT(*) as TotalCompleted",
+            "iterative": true,
+            "iterationsize": 1000
+        },
+        {
+            "query": "MATCH (binding:GCPBinding)<-[a:ATTACHED_BINDING]-(function:GCPFunction)<-[:RESOURCE]-(:GCPProject{id: $GCP_PROJECT_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID})\nWHERE 'allUsers' IN binding.members \nSET function.exposed_internet = true, function.exposed_internet_type = coalesce(function.exposed_internet_type, []) + 'allUsers'",
```

Comment:
to avoid duplicates in `exposed_internet_type ` use like this ` (CASE WHEN NOT 'allUsers' IN  coalesce(instance.exposed_internet_type , []) THEN instance END).exposed_internet_type = coalesce(instance.exposed_internet_type , []) + 'allUsers'`

Reply:
added